app/javascript/src/components/Dashboard/Notes/Card/index.jsx

```diff
@@ -1,20 +1,26 @@
 import React from "react";
 
+import PropTypes from "prop-types";
+
 import CardBody from "./CardBody";
 import CardFooter from "./CardFooter";
 import CardHeader from "./CardHeader";
 
-const Card = () => {
+const Card = ({ setShowDeleteAlert }) => {
   const imageUrl = "https://i.pravatar.cc/300";
   return (
     <>
       <div className="m-2 p-4 border neeto-ui-border-gray-200 neeto-ui-shadow-s">
-        <CardHeader></CardHeader>
+        <CardHeader setShowDeleteAlert={setShowDeleteAlert}></CardHeader>
         <CardBody></CardBody>
         <CardFooter imageUrl={imageUrl}></CardFooter>
       </div>
     </>
   );
 };
 
+Card.propTypes = {
+  setShowDeleteAlert: PropTypes.func
+};
```

Comment:
Pass header, body, and other details as a constant from the parent component. 
Add all of these as prop-types.